Project/Source/Modules/ModuleWindow.cpp

```diff
@@ -128,6 +138,46 @@ void ModuleWindow::SetTitle(const char* title) {
 	SDL_SetWindowTitle(window, title);
 }
 
+void ModuleWindow::SetCursor(bool isPlaying) {
+	if (isPlaying) {
+		// Load resource
+		Scene* scene = App->scene->scene;
+		ResourceTexture* cursorResourceTexture = App->resources->GetResource<ResourceTexture>(scene->GetCursor());
```

Comment:
Been trying to run a scene in Game for a while without success.
Even without a cursor saved for the scene. this lines return an error because 'scene' is nullptr.

I think it has something to do on the order of loading these resources from the Game Configuration.
SetCursor is called in LoadScene() in SceneImporter.cpp. Therefore, the scene is not loaded yet, therefore App.scene.scene has not been created yet.

I think you should be serializing the reference of the texture used, insted of a boolean, and pass it as the parameter of SetCursor(), so you can load the resource directly, without the need to reference the scene.

App->resources->GetResource<ResourceTexture>(referencedResourceId);

Reply:
I'm sorry, I was not understanding what you mean, now it is solved